internal/cli/templates/service/cmd/grpc/main.go.tmpl

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	f "github.com/foundation-go/foundation"
+	ferr "github.com/foundation-go/foundation/errors"
```

Comment:
It's not used in that file. Maybe need to delete

Reply:
It's here just for the demonstration purposes. I don't really think that this is a final version of this template, we deferentially will revise its content in a future. But first we need to figure out what will be useful to have here, I believe.